content/en/docs/reference/workloads-18-19.md

```diff
@@ -1,261 +0,0 @@
----
```

Comment:
Why was this content removed?

Reply:
Long out of date. We've mentioned it once or twice in sig meetings -- that it was getting to be time to yank it from docs versions where it is no longer relevant.

Reply:
Awesome! Sounds great!